src/main/java/org/jabref/gui/groups/GroupTreeViewModel.java

```diff
@@ -205,7 +207,7 @@ public void addSuggestedSubGroup(GroupNodeViewModel parent) {
 
             if (!hasEntriesWithoutFiles) {
                 SearchGroup withoutFilesGroup = new SearchGroup(
-                        Localization.lang("Entries without linked files"),
+                        Localization.lang(ENTRIES_WITHOUT_LINKED_FILES),
```

Comment:
This is now how it works https://devdocs.jabref.org/code-howtos/localization.html#general-hints